app/src/main/java/ie/macinnes/tvheadend/sync/EpgSyncTask.java

```diff
@@ -352,6 +353,7 @@ private void handleChannelAddUpdate(@NonNull HtspMessage message) {
             ));
         }
 
+        //TODO: Why do these both do the same thing?
```

Comment:
They don't do the same thing :)

The first branch (where `mInitialSyncCompleted == true`) will - once the initial sync has completed - flush every channel operation as it comes in.

The second branch (!IS_BRAVIA && mPendingChannelOps.size() >= 100) will - until the initial sync has completed - flush every 100 channel operations. Unless you're on a Sony Bravia, in that case, we wait for every channel to arrive before flushing so we can sort them correctly to workaround a Sony issue.